apps/expo/package.json

```diff
@@ -51,10 +51,13 @@
     "react-dom": "18.2.0",
     "react-native": "0.73.6",
     "react-native-body-highlighter": "3.0.0-beta.6",
+    "react-native-chart-kit": "^6.12.0",
```

Comment:
No longer needed for my chart. Can remove.

Reply:
can you remove the extra branch as well

Reply:
Extra branch?

Reply:
yes, extra branch

Reply:
![Screenshot_20240424-150535](https://github.com/UNLV-CS472-672/2024-S-GROUP3-Barbell/assets/94078395/66b6a720-07f6-40ac-a113-390e415a3d64)

Reply:
Done I believe.